apps/kaze/app/(main)/[slug]/page.tsx

```
@@ -48,18 +48,18 @@ export default async function ProductDetailPage({
   return (
     <HydrateClient>
       <script
-        type="application/ld+json"
+        type='application/ld+json'
         // eslint-disable-next-line @eslint-react/dom/no-dangerously-set-innerhtml
         dangerouslySetInnerHTML={{ __html: JSON.stringify(jsonLd) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)